app/src/main/java/com/example/geoquiz/MainActivity.kt

```diff
@@ -47,6 +51,12 @@ class MainActivity : AppCompatActivity() {
             Toast.makeText(this, quizViewModel.checkAnswer(false), Toast.LENGTH_SHORT).show()
         }
 
+        cheatButton.setOnClickListener {
+            val answerQuestion = quizViewModel.currentQuestionAnswer
+            val intent = CheatActivity.newIntent(this@MainActivity, answerQuestion)
+            startActivityForResult(intent, REQUEST_CODE_CHEAT)
```

Comment:
@iugaliviu mentioned that `startActivityForResult` is deprecated
Please use the new method:  https://stackoverflow.com/questions/62671106/onactivityresult-method-is-deprecated-what-is-the-alternative